packages/vats/tools/fake-bridge.js

```diff
@@ -157,6 +157,48 @@ export const makeFakeIbcBridge = (zone, onToBridge) => {
 
 export const LOCALCHAIN_DEFAULT_ADDRESS = 'agoric1fakeLCAAddress';
 
+/**
+ * Used to mock responses from Cosmos Golang back to SwingSet for for
+ * E(lca).executeTx().
+ *
+ * Returns an empty object per message unless specified.
+ *
+ * @param {object} message
+ * @param {number} sequence
+ * @returns {unknown}
+ * @throws {Error} to simulate failures in certain cases
+ */
+export const fakeLocalChainBridgeTxHandler = (message, sequence) => {
+  switch (message['@type']) {
+    // TODO #9402 reference bank to ensure caller has tokens they are transferring
+    case '/ibc.applications.transfer.v1.MsgTransfer': {
+      if (message.token.amount === '504') {
+        throw Error('simulated unexpected MsgTransfer packet timeout');
+      }
+      // like `JsonSafe<MsgTransferResponse>`, but bigints are converted to numbers
```

Comment:
Good question. I would like to observe the behavior in an E2E environment and report back. It's been some time since this was written but iirc it was motivated by observed values.

Can we add an XXX/TODO for now? `LCA.transfer()` is out of scope here, but I expect to add E2E tests for it as part of the ticket this PR refs.

Reply:
Ok to punt it but I think it should be FIXME since it doesn't match the type declaration